api/v1alpha1/serverless_types.go

```diff
@@ -32,9 +32,16 @@ type Endpoint struct {
 
 // ServerlessSpec defines the desired state of Serverless
 type ServerlessSpec struct {
-	Tracing        *Endpoint       `json:"tracing,omitempty"`
-	Eventing       *Endpoint       `json:"eventing,omitempty"`
-	DockerRegistry *DockerRegistry `json:"dockerRegistry,omitempty"`
+	Tracing                  *Endpoint       `json:"tracing,omitempty"`
+	Eventing                 *Endpoint       `json:"eventing,omitempty"`
+	DockerRegistry           *DockerRegistry `json:"dockerRegistry,omitempty"`
+	CPUUtilizationPercentage string          `json:"CPUUtilizationPercentage,omitempty"`
```

Comment:
Should those field has prefix like `function`, according to the issue: https://github.com/kyma-project/serverless-manager/issues/105